nemo_text_processing/hybrid/mlm_scorer.py

```diff
@@ -0,0 +1,100 @@
+# Copyright (c) 2022, NVIDIA CORPORATION & AFFILIATES.  All rights reserved.
+# Copyright 2020 AWSLABS, AMAZON.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import List
+
+import numpy as np
+
+try:
+    import torch
+    from torch.nn.functional import softmax
+except ImportError as e:
+    raise ImportError("torch is not installed")
```

Comment:
have you tried this in a clean env without torch?

Reply:
yes, it just throws an error, even wiithout the exception it does actually, i think it is fine